stories/Preloader.stories.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import Row from '../src/Row';
+import Col from '../src/Col';
+import ProgressBar from '../src/ProgressBar';
+import Preloader from '../src/Preloader';
+
+const stories = storiesOf('components/Preloader', module);
```

Comment:
`const stories = storiesOf('Components|Preloader', module);` please

Reply:
Done :)

Added knobs to progress props too.